libraries/SITL/SITL_Input.h

```diff
@@ -7,7 +7,7 @@
   microseconds
 */
 struct sitl_input {
-    uint16_t servos[16];
+    uint16_t servos[32];
```

Comment:
How about we pull from "static const uint8_t max_channels"

Reply:
that is in ChibiOS and this is SITL